src/types/privatekubectl.ts

```diff
@@ -0,0 +1,121 @@
+import { Kubectl } from "./kubectl";
+import { ExecOptions, ExecOutput, getExecOutput } from "@actions/exec";
+import * as core from "@actions/core";
+import * as os from "os";
+import * as fs from "fs";
+
+
+
+export class PrivateKubectl extends Kubectl{
+
+
+  public isPrivate(): boolean {
+      return true;
+  }
+
+  protected async execute(args: string[], silent: boolean = false) {
+    args.unshift("/k8stools/kubectl");
+    var kubectlCmd = args.join(" ");
```

Comment:
use `let` instead of `var` everywhere.

Reply:
Git has been acting weird lately. `let` is being used now, but github continues to show outdated code.